### PR TITLE
Update to Panel

### PR DESCRIPTION
**Pull request by zeth**

### CODE REVIEW

File: content/abstracts.rst [hunk 1]
```diff
@@ -114,6 +114,10 @@ The PyCon UK panel will probably include:
  * Larry Hastings is one of the main CPython core contributors and the
    Release Manager for Python releases.
 
+ * Nicholas Tollervey is a Python trainer, former teacher, organiser
```
Comment (ntoll) [inline content/abstracts.rst]: Nicholas Tollervey is a freelance Python developer, classically trained musician, philosophy graduate, teacher, writer, organiser of the education track, founder of the London Python Code Dojo and PSF ambassador to the BBC micro:bit project. He enjoys spending his copious free time playing the tuba or hacking on distributed hash tables.

**Comment by ntoll** [comment]
Please change the bio. Otherwise, oh, go on then... ;-)

**Comment by zeth** [comment]
New bio added, but also:
- A currently unselected Djangogirls organiser

Do we have one yet?

**Comment by ntoll** [comment]
Ask both of the Olas (Peter knows them better than I do)

File: content/abstracts.rst [hunk 2]
```diff
@@ -108,13 +108,21 @@ The PyCon UK panel will probably include:
...
 
- * A currently unselected Djangogirls organiser
+ * Nicholas Tollervey: freelance Python developer, classically
+   trained musician, philosophy graduate, teacher, writer, organiser of
+   the education track, founder of the London Python Code Dojo and PSF
+   ambassador to the BBC micro:bit project. He enjoys spending his
+   copious free time playing the tuba or hacking on distributed hash
+   tables.
+
+ * Ola Sendecka: Python and Django enthusiast. Django Girls
```
Comment (inglesp) [inline content/abstracts.rst]: @zeth, has Ola replied?  Don't think we should add people to this list who haven't said yes yet.

**Comment by zeth** [comment]
This is mergeable, but the drogulus is too popular to kill.

**Comment by inglesp** [comment]
Merged -- thanks for the prod!

**Pull request merged**